packages/framework-core/src/booster-events-reader.ts

```diff
@@ -0,0 +1,78 @@
+import {
```

Comment:
This is the brand new class that allow us to read events. It will be used by GraphQL and `Booster.events` to be able to read events